k8s/namespaces/private-law/prl-citizen-frontend/demo.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: prl-citizen-frontend
+  namespace: private-law
+spec:
+  releaseName: prl-citizen-frontend
+  rollback:
+    enable: true
+    retry: true
+  chart:
+    git: git@github.com:hmcts/hmcts-charts
+    ref: master
+    path: stable/prl-citizen-frontend
+  values:
+    java:
+      replicas: 2
```

Comment:
replicas should be set in the base file not env specific unless you're overriding

Reply:
base file contains the replicas, so removed duplicate config